components/TaskList/TaskList.jsx

```diff
@@ -1,20 +1,42 @@
 const React = require('react')
+const { connect } = require('react-redux')
 
-class TaskList extends React.Component {
-  render() {
-    return (
-      <div>
-        <nav>
-          <button>snooze</button>
-          <button>ignore</button>
-          <button>archive</button>
-        </nav>
-        <ol>
-          <li>LOOK AT THIS ISSUE</li>
-        </ol>
-      </div>
-    )
+const TaskList = ({tasks}) => {
+  const taskElements = tasks
+    .filter((task) => {
```

Comment:
I'm following the AirBnb style guide!

`Use indentation when making long method chains. Use a leading dot, which emphasizes that the line is a method call, not a new statement.`